cmake/quickjs.cmake

```diff
@@ -30,7 +30,7 @@ if("sgx" IN_LIST COMPILE_TARGETS)
   target_compile_options(
     quickjs.enclave
     PUBLIC -nostdinc -DCONFIG_VERSION="${QUICKJS_VERSION}" -DEMSCRIPTEN
-           -DCONFIG_STACK_CHECK
+           -DCONFIG_STACK_CHECK -DCONFIG_BIGNUM
```

Comment:
This used to bring in libgmp iirc, which I why I'd turned it off. If it works happily now, great!